lineage/figures/figureCommon.py

```diff
@@ -15,16 +15,18 @@
 pi = np.array([0.5, 0.5], dtype="float")
 
 # T: transition probability matrix
-T = np.array([[0.95, 0.05], [0.05, 0.95]], dtype="float")
+T = np.array([[0.9, 0.1], [0.1, 0.9]], dtype="float")
 
 # bern, gamma_a, gamma_scale
-state0 = StateDistribution(0.99, 20, 5)
-state1 = StateDistribution(0.75, 10, 1)
+state0 = StateDistribution(0.99, 7, 7)
```

Comment:
new fixed gamma parameter values,
shape value is constant,
scale value is varying